### ISSUE TITLE
Missing Get-SFCommand after Install-Module

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/PowerShell#38]
If v1.5 is installed using `Install-Module`, some commandlets such as `Get-SFCommand` (maybe there are others) are missing.

```
PS C:\Users\Sean> Get-Module -Name SolidFire

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Manifest   1.5.0.119  solidfire                           {Add-SFDrive, Add...


PS C:\Users\Sean> $psversiontable.psversion

Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      16299  98


PS C:\Users\Sean> Get-SFCommand
Get-SFCommand : The term 'Get-SFCommand' is not recognized as the name of a
cmdlet, function, script file, or operable program. Check the spelling of the
name, or if a path was included, verify that the path is correct and try
again.
At line:1 char:1
+ Get-SFCommand
+ ~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Get-SFCommand:String) [], Comm
   andNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

```

When I use the old school approach (MSI installer, or steps from https://github.com/solidfire/PowerShell/blob/1.4/docs/linux/README.md#next-install-solidfire-bits using current versions of PS and SF Tools for PS), the command is still there and works fine.

**Comment by ahaid** [comment]
Get-SFCommand is only available because of the initialization script that gets registered during the wizard installation of the .msi. Under the hood, it is calling `Get-Command -Module SolidFire` which is a more powerrshelly way to discover methods in a module. I decided to push people in that direction for the non-installer versions of our module. See the instructions here: https://github.com/solidfire/PowerShell/blob/master/docs/mac/README.md.

**Comment by scaleoutsean** [comment]
Ok, makes sense. 

To recap my case:

* I upgraded from 1.4.0 with the init script
* After installing the new way, my old init script still kicked in after `pwsh`
* I trusted the init screen (now showing the latest & greatest module version) that told me `Get-SFCommand` is available. 

![image](https://user-images.githubusercontent.com/16622642/33569413-97406956-d964-11e7-8239-c04fe43f34e2.png)

It's certainly not a big deal, but it does mean that some users may get confused during transition from MSI package to module installation. I'll close this as I already have one issue on install/uninstall process.

I read the link, but I didn't click on the Gallery link. Now I did and it's wrong, so I'll submit a PR for that.
